Running container fails on GOPATH \w multiple dirs entries

Default GOPATH on GCE cloud console is `"/home/alex/gopath:/google/gopath"`

But running a build docker container for any driver fails on [mounting such GOPATH](https://github.com/bblfsh/sdk/blob/master/etc/build/make/rules.mk#L43)

```
+ docker run --rm -t -u bblfsh:1000 -v /home/alex/java-driver:/opt/driver/src/ -v /home/alex/gopath:/google/gopath:/go -e ENVIRONMENT=bblfs
h/java-driver-build-with-go bblfsh/java-driver-build-with-go make test-driver-internal

docker: Error response from daemon: Invalid bind mount spec "/home/alex/gopath:/google/gopath:/go": invalid mode: /go.
See 'docker run --help'.
```

This also breaks the driver builds i.e in case of python

```
docker: Error response from daemon: Invalid bind mount spec "/home/alex/gopath:/google/gopath:/go": invalid mode: /go.
See 'docker run --help'.
/home/alex/gopath/src/github.com/bblfsh/python-driver/.sdk/make/rules.mk:101: recipe for target 'build-driver' failed
make: *** [build-driver] Error 125
```

--- a/assets/build/bindata.go
+++ b/assets/build/bindata.go
@@ -496,6 +496,8 @@ GO_CMD = go
 GO_RUN = $(GO_CMD) run
 GO_TEST = $(GO_CMD) test -v
 GO_LDFLAGS = -X main.version=$(DRIVER_VERSION) -X main.build=$(BUILD_ID)
+# If GOPATH has multiple dir entries, pick the first one
+GO_PATH := $(shell echo ${GOPATH}|cut -f1 -d':')
 
 # build enviroment variables
 BUILD_USER ?= bblfsh
@@ -511,7 +513,7 @@ BUILD_NATIVE_CMD ?= $(DOCKER_RUN) \
 BUILD_DRIVER_CMD ?= $(DOCKER_RUN) \
 	-u $(BUILD_USER):$(BUILD_UID) \
 	-v $(BUILD_VOLUME_PATH):$(BUILD_VOLUME_TARGET) \
-	-v $(GOPATH):/go \
+	-v $(GO_PATH):/go \
 	-e ENVIRONMENT=$(DOCKER_BUILD_DRIVER_IMAGE) \
 	-e HOST_PLATFORM=$(shell uname) \
 	$(DOCKER_BUILD_DRIVER_IMAGE)
@@ -610,7 +612,7 @@ func makeRulesMk() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "make/rules.mk", size: 4060, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
+	info := bindataFileInfo{name: "make/rules.mk", size: 4167, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
 	a := &asset{bytes: bytes, info: info}
 	return a, nil
 }

--- a/etc/build/make/rules.mk
+++ b/etc/build/make/rules.mk
@@ -27,6 +27,8 @@ GO_CMD = go
 GO_RUN = $(GO_CMD) run
 GO_TEST = $(GO_CMD) test -v
 GO_LDFLAGS = -X main.version=$(DRIVER_VERSION) -X main.build=$(BUILD_ID)
+# If GOPATH has multiple dir entries, pick the first one
+GO_PATH := $(shell echo ${GOPATH}|cut -f1 -d':')
 
 # build enviroment variables
 BUILD_USER ?= bblfsh
@@ -42,7 +44,7 @@ BUILD_NATIVE_CMD ?= $(DOCKER_RUN) \
 BUILD_DRIVER_CMD ?= $(DOCKER_RUN) \
 	-u $(BUILD_USER):$(BUILD_UID) \
 	-v $(BUILD_VOLUME_PATH):$(BUILD_VOLUME_TARGET) \
-	-v $(GOPATH):/go \
+	-v $(GO_PATH):/go \
 	-e ENVIRONMENT=$(DOCKER_BUILD_DRIVER_IMAGE) \
 	-e HOST_PLATFORM=$(shell uname) \
 	$(DOCKER_BUILD_DRIVER_IMAGE)

